modules/service/src/main/scala/lucuma/odb/graphql/mapping/ExecutionMapping.scala

```diff
@@ -58,7 +60,7 @@ trait ExecutionMapping[F[_]] extends ObservationEffectHandler[F]
     ObjectMapping(ExecutionType)(
       SqlField("id", ObservationView.Id, key = true, hidden = true),
       SqlField("programId", ObservationView.ProgramId, hidden = true),
-      EffectField("digest", digestHandler, List("id", "programId")),
+      SqlObject("digest", Join(ObservationView.Id, ObscalcView.Digest.Id)),
```

Comment:
Here is where the old compute-on-demand effect handler is swapped out for a normal `SqlObject` mapping.